core/root/template_params.go

```diff
@@ -16,6 +16,10 @@ func (p TemplateParams) ClusterName() string {
 	return p.cluster.controlPlane.ClusterName
 }
 
+func (p TemplateParams) ClusterVersion() string {
```

Comment:
So that we won't confuse this with k8s version? Makes sense! I'll make a follow-up PR for it.